java-squid/src/main/java/org/sonar/java/ast/visitors/PublicApiChecker.java

```diff
@@ -75,6 +75,13 @@
   private final Deque<Tree> currentParents = new LinkedList<Tree>();
   private double publicApi;
   private double documentedPublicApi;
+  private boolean analyseAccessors;
+  private final AccessorVisitor accessorVisitor;
+
+  public PublicApiChecker(boolean analyseAccessors) {
```

Comment:
Boolean trap : http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html
Having a constructor with a boolean parameter does not seem the best option here. Maybe two methods to create the instance with correct setting and this constructor goes private.